src/content/advanced/storage/azure-pvc-encryption/index.md

```diff
@@ -0,0 +1,79 @@
+---
+linkTitle: Persistent Volumes data encryption with Azure Key Vault
```

Comment:
`linkTitle` is the one displayed in the menu. It's a bit too long.

![image](https://user-images.githubusercontent.com/273727/122889680-49d69000-d343-11eb-88d4-e7714ffeb8d1.png)

I suggest something like `Azure volume encryption` here.